plugins/sentinelone/plugin.spec.yaml

```diff
@@ -19,1445 +21,1669 @@ tags:
 - detection
 hub_tags:
   use_cases: [threat_detection_and_response]
-  keywords: [sentinelone, endpoint, detection]
+  keywords: [sentinelone, endpoint, detection, cloud_enabled]
   features: []
 types:
-  activities_types:
+  activityTypes:
```

Comment:
Why did we change the casing of our inputs?

Reply:
Since all fields in the output of each action are in camelCase, to maintain consistency in the plugin we also changed the naming of the inputs to camelCase.